templates-extension.md

```diff
@@ -409,195 +467,182 @@ The following [JSON schema](https://datatracker.ietf.org/doc/html/draft-bhutton-
+1. The application identifies the source template (if any) from the `_TPLT` of the master source.
+2. The application prompts the user to enter values for citation elements as specified by the `dfields` array in the source template.
+3. The application sets the `PAGE` value by generated using the `page` template pattern in the source template.  If a template pattern is not specified in the source template, the GEDCOM tag is omitted.
+4. The application generates text for the citation using the `first` template pattern.  If such a pattern is not present, the following template pattern is used instead: `{% if scite %}{{ scite }}{% else %}{{ TITL }}{% if AUTH %}, by {{ AUTH }}{% endif %}{% if PUBL %} ({{ PUBL }}){% endif %}{% endif %}{% if PAGE %}, {{ PAGE }}{% endif %}{% if URL %}; {{ URL }}{% endif %}{% if NOTE %}. {{ NOTE }}{% endif %}.`  When evaluating this template pattern, the application substitutes values as follows:
+   * Any field appearing in the source template (whether in the `sfields` or `dfields` arrays) uses the value entered by the user.
+   * If the pattern `scite` appears in the source template, it is expanded using the template pattern there.
+   * Otherwise variables are expanded using the value of the corresponding GEDCOM tag.
+5. The application creates `_FIEL` substructures of the `PAGE` with variables and values of any citation elements that (a) are in the `dfields` of the source template, and (b) have non-empty values.
 
-**`SOUR` (Collection)**  `<ext:_TPLT-SOUR>`
+## Examples of Source Templates
 
-A source record for the collection in which the containing record is present.
+The following examples show the YAML text, which can be placed into `.yaml` files.
 
-And the following additions to the existing `SOURCE_RECORD` and `SOURCE_CITATION` definitions:
+### Grave Marker Template
 
-`SOURCE_RECORD` :=
-```gedstruct
-n @XREF:SOUR@ SOUR                      {1:1}  g7:record-SOUR
-  ...
-  +1 <<SOURCE_MODEL_REFERENCE>>         {0:1}
-```
+The Grave Marker example is taken from the monument example at *Evidence Explained*, page 247.
 
-`SOURCE_CITATION` :=
-```
-n SOUR @<XREF:SOUR>@                    {1:1}  g7:SOUR
-   ...
-  +1 PAGE <Text>                        {0:1}  g7:PAGE
-      ...
-     +2 <<CITATION_ELEMENT>>            {0:M}
-```
-
-### Examples
-
-#### Grave Marker Template
-
-The Grave Marker example is taken from the example at *Evidence Explained*, pages 213-214.
-
-##### YAML
+#### Grave Marker YAML
 
 ```yaml
 %YAML 1.2
 ---
 lang: en-US
 uri: https://gedcom.io/templates/GraveMarker
 name: Grave Marker
-description: The Grave Marker model is used for tombstones and other types of
-  grave markers (see "Evidence Explained", pages 213-214).  It is not used for
-  digital images found online; for those use the Website model instead.
+description: The Grave Marker model is used for grave markers, cemetery monuments,
+  and other forms of cemetery art (see "Evidence Explained", pages 246-247).
```

Comment:
I have the 2007 edition and pages 246-247 are about census population schedules.